meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: tomopy
-  version: '0.1.8'
+  version: '0.1.9'
 
 source:
   git_url: https://github.com/dgursoy/tomopy.git
```

Comment:
There are 3 places you need to keep track of the version: meta.yaml,
setup.py, and __version__ in the top level `__init__.py`.  Of the three I
think getting the conda one right is most important (so that your
non-release builds sort right and so you can always get back _exactly_ what
a given version was), then the `__version__` as that is what users tend to
see and last the version in `setup.py` as it is only used by pip and I only
marginally care about supporting that.

The way I normally deal with that is to between releases change the version
in `setup.py` and `__init__.py` to `vX.Y.Z.post0`  so it indicates that it
isn't an 'official' release but is some intermediate after `vX.Y.Z` and let
the conda packages keep the real version.  There is a tool (versioneer)
which will handle all of this automatically, @danielballan has experience
with it, but I have not had the bandwidth to get it set up the way I want.



On Mon, May 18, 2015 at 7:31 PM Doga Gursoy <notifications@github.com>
wrote:

> In meta.yaml
> <https://github.com/tomopy/tomopy/pull/89#discussion_r30557287>:
>
> > @@ -1,6 +1,6 @@
> >  package:
> >    name: tomopy
> > -  version: '0.1.8'
> > +  version: '0.1.9'
> >
> >  source:
> >    git_url: https://github.com/dgursoy/tomopy.git
>
> I like the idea but shouldn't it be better if it is the same as the one
> described in setup.py?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tomopy/tomopy/pull/89/files#r30557287>.
>